src/tile/tile.scss

```diff
@@ -175,9 +175,7 @@
       }
 
       > .iui-name-label {
-        text-overflow: ellipsis;
-        overflow: hidden;
-        white-space: nowrap;
+        @include iui-line-clamp(1);
```

Comment:
I think we need `@supports` with a fallback to ellipsis for IE.

Reply:
the mixin already has that